src/util/mq.ts

```diff
@@ -0,0 +1,17 @@
+export enum BreakPoint {
+  sm,
+  md,
+  lg,
+  xl,
+}
+
+const screenSizes = {
+  [BreakPoint.sm]: 640,
+  [BreakPoint.md]: 768,
+  [BreakPoint.lg]: 1024,
+  [BreakPoint.xl]: 1280,
+};
+
+export const mq = (size: BreakPoint) => {
+  return `@media (min-width: ${screenSizes[size]}px)`;
```

Comment:
I'd add `screen and` here so as not to mess with `print` and `speech` representations